test/Microsoft.Identity.Abstractions.Tests/AquireTokenOptionsTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Xunit;
+
+namespace Microsoft.Identity.Abstractions.Tests
+{
+    public class AquireTokenOptionsTests
+    {
+        [Fact]
+        public void ManagedIdentitySystemAssigned()
+        {
+            // App token from a system-assigned managed identity.
+            // -------------------------------------------------
+            // https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview
+            /*
+            // <managedidentity_json>
+            {
+                "AquireTokenOptions": {
+                    "ManagedIdentity": {
+                        "Source": "SystemAssigned"
+                    }
+                }
+            }
+            // </managedidentitysystem_json>
+            */
+
+            // <managedidentity_csharp>
+            ManagedIdentityDescription managedIdentityDescription = new ManagedIdentityDescription
+            {
+                Source = ManagedIdentitySource.SystemAssigned
+            };
+
+            AcquireTokenOptions acquireTokenOptions = new AcquireTokenOptions
+            {
+                ManagedIdentity = managedIdentityDescription
```

Comment:
this is verbose.

Developers would like to write:

```csharp
 AcquireTokenOptions acquireTokenOptions = new AcquireTokenOptions
            {
                ManagedIdentity = ManagedIdentityDescription.SystemAssigned
           }
```

where `ManagedIdentityDescription.SystemAssigned` would be a static get property returning an already initialized version

Reply:
If you look at the implementation this is possible as I do default any instance to SystemAssigned, I was thinking the verbosity was useful in aiding comprehension/readability, many customers may not know there are two kinds of managed identity, and this will help clue them in not only to the different kinds but also which one their code is using.

Reply:
It's a good philosophy, @JoshLozensky 
but
In IdWeb, we tend to use another which philosophy is "pay only for what you need". Or "simple things should be simple. More complex things should be possible with small incremental effort".
The whole library is like that, and FIC with MSI (or AKS) is on this principle.